### DISCUSSION TITLE
How to setup log

### DISCUSSION THREAD

**Discussion by kolserdav** [docker-mailserver/docker-mailserver#4027] [Q&A]
Hello everyone!
I receive flooding logs while operational system is staring:
```sh
2024-05-21T00:36:48.187376+00:00 mail postfix/tlsmgr[22616]: warning: end-of-input while reading request from tlsmgr socket: Application error

```
How can I set up logs to hide this messages?

**Comment by glime-ai** [comment]
**Please note** that this response was created using generative AI, leveraging previously available information from the repository. This information may be helpful. However, it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Hello!

The message you're seeing in the logs is a warning from Postfix's `tlsmgr` (TLS manager) service. It indicates that there was an unexpected end-of-input while reading a request, which could be due to various reasons including configuration issues or network problems.

To reduce the verbosity of such messages in your logs, you can adjust the logging level of Postfix. However, it's important to note that completely hiding these messages is not recommended as they can provide valuable insights into potential issues within your mail server setup. Instead, you might want to adjust the severity level at which logs are recorded.

Here's how you can adjust the logging level:

1. **Modify Postfix's main configuration**:
   You can set the `log_level` parameter in Postfix's main configuration file (`main.cf`). Lower numbers are more verbose. Setting it to a higher number can reduce the verbosity. For example:
   ```bash
   postconf -e "log_level = 2"
   ```

2. **Restart Postfix to apply the changes**:
   After making changes to the configuration, restart Postfix to apply the new settings:
   ```bash
   postfix reload
   ```

3. **Monitor the logs**:
   After adjusting the log level, continue to monitor your logs to ensure that you are not missing critical warnings or errors.

If you are using Docker Mailserver, you might also want to check the environment variable `LOG_LEVEL` which can be set to control the verbosity of the logs at the Docker container level. You can find more details on setting environment variables in the Docker Mailserver documentation.

Remember, adjusting log levels should be done with caution, especially in a production environment, as it might suppress important warnings that could help you diagnose and resolve issues.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=0d5b86fb7eea4a54acac405212c29aba&utm_medium=github).
